src/icon-tab-bar.scss

```diff
@@ -320,6 +334,21 @@ $fd-icon-tab-bar-semantic-values: (
         }
       }
     }
+
+    &--hidden {
+      display: none;
+    }
+
+    &--extra-btn {
```

Comment:
This is unused too

Reply:
Remove this class and applied its styles for --overflow modifier